src/js/lib/content/src/index.ts

```diff
@@ -955,20 +955,25 @@ API.main = async function(inputConfig: any) {
     config.updateFromObject(inputConfig)
     config.updateFromObject(urlConfig)
 
+    const runEntryPoint = async () => {
```

Comment:
Can this not be written as a `function` instead of an alignment of a closure?

Reply:
Yes, I refactored it as a function :+1: